src/sentry/discover/endpoints/discover_key_transactions.py

```diff
@@ -24,6 +24,32 @@ class KeyTransactionPermission(OrganizationPermission):
     }
 
 
+class LegacyKeyTransactionCountEndpoint(KeyTransactionBase):
```

Comment:
to check, this is to make EA easier right? my initial reaction is we could just re-use the existing `KeyTransactionEndpoint` but I might be missing something here.

Reply:
Unsure how we would use the existing `KeyTransactionEndpoint` for this.

To clarify, this endpoint should be temporary during the rollout of team key transactions to make users that previously had any key transactions (not just a particular key transaction) aware that their old key transactions are gone.

Reply:
My mistake, thought we had an endpoint that already returns the count of KT. :+1: that explanation of the endpoint makes sense